src/go/rpk/pkg/cli/cmd/common/common.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2020 Vectorized, Inc.
```

Comment:
NIT: It's 2021

Reply:
2021 doesn't exist, it's just a very long 2020 haha. Good catch!